common/changes/@itwin/tree-widget-react/tree-widget-external-sources-tree_2023-04-07-13-59.json

```diff
@@ -0,0 +1,15 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/tree-widget-react",
+      "comment": "Add `ExternalSourcesTreeComponent` (currently `@alpha`).",
+      "type": "patch"
```

Comment:
im not 100% certain that this will work as intended with the pre-release sutff, i need to look into this more tomorrow

Reply:
appui, hopefully, won't take long to release 4.0 - I think this PR can wait for that to happen and merge after we upgrade to 4.0 appui and presentation. In that case no pre-releases would be necessary.